Stride.BepuPhysics/Components/Colliders/TriangleColliderComponent.cs

```diff
@@ -1,15 +1,21 @@
-﻿using Stride.BepuPhysics.Processors;
+﻿using BepuPhysics;
+using BepuPhysics.Collidables;
+using Stride.BepuPhysics.Extensions;
+using Stride.BepuPhysics.Processors;
 using Stride.Core;
 using Stride.Core.Mathematics;
 using Stride.Engine;
 using Stride.Engine.Design;
+using Stride.Games;
+
+#warning I don't think this would have any actual use, you can keep this internal if you want to keep it for debugging purposes
```

Comment:
Possibly for 2d? This could be inherited to build more shapes but I could be wrong.

Reply:
I think meshes already cover that particular use case